test/transactions/multisignature.js

```diff
@@ -1,7 +1,166 @@
+		var createMultisig = multisignature.createMultisignature('secret', '', multiSignaturePublicKeyArray, requestLifeTime, minimumSignatures);
+
+		it('should create Multisignature account with single secret', function () {
+			(createMultisig).should.be.ok;
+			(createMultisig).should.be.type('object');
+		});
+
+		it('should be transaction type 4', function () {
+			(createMultisig.type).should.be.equal(4);
+		});
+
+		it('should have no recipient', function () {
+			expect(createMultisig.recipientId).to.be.null;
+		});
+
+		it('should contain asset with multisignature and the inserted parameters', function () {
+			(createMultisig.asset.multisignature.min).should.be.equal(minimumSignatures);
+			(createMultisig.asset.multisignature.lifetime).should.be.equal(requestLifeTime);
+			(createMultisig.asset.multisignature.keysgroup).should.be.equal(multiSignaturePublicKeyArray);
+		});
+
+		it('should not contain secondSignature', function () {
+			expect(createMultisig.signSignature).to.be.undefined;
+		});
+	});
+
+	describe('#createMultisignature with two secrets', function () {
+
+		var minimumSignatures2 = 6;
+		var requestLifeTime2 = 8;
+		var multiSignaturePublicKeyArray2 = ['+123456789', '+1236345489', '+123452349', '-987654321', '+12323432489','+1234234789', '-82348375839'];
+		var createMultisig2 = multisignature.createMultisignature('secret', 'secondSecret', multiSignaturePublicKeyArray2, requestLifeTime2, minimumSignatures2);
+
+		it('should create Multisignature account with two secrets', function () {
+			(createMultisig2).should.be.ok;
+			(createMultisig2).should.be.type('object');
+		});
+
+		it('should be transaction type 4', function () {
+			(createMultisig2.type).should.be.equal(4);
+		});
+
+		it('should have no recipient', function () {
+			expect(createMultisig2.recipientId).to.be.null;
+		});
+
+		it('should contain asset with multisignature and the inserted parameters', function () {
+			(createMultisig2.asset.multisignature.min).should.be.equal(minimumSignatures2);
+			(createMultisig2.asset.multisignature.lifetime).should.be.equal(requestLifeTime2);
+			(createMultisig2.asset.multisignature.keysgroup).should.be.equal(multiSignaturePublicKeyArray2);
+		});
+
+		it('should contain secondSignature', function () {
+			expect(createMultisig2.signSignature).not.to.be.undefined;
+		});
+	});
+
+	describe('#signTransaction', function () {
+
+		var secret = '123';
+		var transaction = lisk.transaction.createTransaction('58191285901858109L', 1000, 'secret');
+		var signTransaction = multisignature.signTransaction(transaction, secret);
+
+		it('should sign a transaction', function () {
+			(signTransaction).should.be.ok;
+		});
+
+		it('should be a string', function () {
+			(signTransaction).should.be.type('string');
+		});
+
+		it('should be crypto_sign_BYTES length', function () {
+			var length = 128;
+			(signTransaction).should.have.lengthOf(length);
+		});
+
+		it.skip('should be verifiable', function () {
```

Comment:
yes.
This test will be resolved (or tackled) in the Issue: https://github.com/LiskHQ/lisk-js/issues/88